<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserRoleManagementDialog</class>
 <widget class="QDialog" name="UserRoleManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>860</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User Role Management</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/plugins/lm2/user_role_management.png</normaloff>:/plugins/lm2/user_role_management.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>39</y>
     <width>331</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>User Roles</string>
   </property>
   <widget class="QLineEdit" name="username_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Retype Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="retype_password_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>182</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC Address(es)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mac_address_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>15</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstname_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>55</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>133</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>213</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_button">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>213</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>213</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Existing User Roles</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>95</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="user_role_lwidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>311</width>
      <height>106</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="position_cbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>39</y>
     <width>230</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Groups</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Groups Available</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down_groups_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>187</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/plugins/lm2/down_button.png</normaloff>:/plugins/lm2/down_button.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="up_groups_button">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>187</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/plugins/lm2/up_button.png</normaloff>:/plugins/lm2/up_button.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>209</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Groups the user is a member of</string>
    </property>
   </widget>
   <widget class="QListWidget" name="group_lwidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>126</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="member_lwidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>245</y>
      <width>211</width>
      <height>126</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>798</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Role Management</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>39</y>
     <width>204</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Spatial Restrictions</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>25</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aimag / Duureg</string>
    </property>
   </widget>
   <widget class="QComboBox" name="aimag_cbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>184</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="aimag_lwidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>109</y>
      <width>184</width>
      <height>76</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="down_aimag_button">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>65</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/plugins/lm2/down_button.png</normaloff>:/plugins/lm2/down_button.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="up_aimag_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>65</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/plugins/lm2/up_button.png</normaloff>:/plugins/lm2/up_button.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>195</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Soum</string>
    </property>
   </widget>
   <widget class="QComboBox" name="soum_cbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>184</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="soum_lwidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>184</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="up_soum_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>235</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/plugins/lm2/up_button.png</normaloff>:/plugins/lm2/up_button.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="down_soum_button">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>235</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/plugins/lm2/down_button.png</normaloff>:/plugins/lm2/down_button.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Granted Aimags / Duuregs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>265</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Granted Soums</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>703</x>
     <y>430</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="help_button">
   <property name="geometry">
    <rect>
     <x>602</x>
     <y>430</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/lm2/help_button.png</normaloff>:/plugins/lm2/help_button.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>799</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username_edit</tabstop>
  <tabstop>password_edit</tabstop>
  <tabstop>retype_password_edit</tabstop>
  <tabstop>mac_address_edit</tabstop>
  <tabstop>surname_edit</tabstop>
  <tabstop>firstname_edit</tabstop>
  <tabstop>phone_edit</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>update_button</tabstop>
  <tabstop>delete_button</tabstop>
  <tabstop>user_role_lwidget</tabstop>
  <tabstop>group_lwidget</tabstop>
  <tabstop>down_groups_button</tabstop>
  <tabstop>up_groups_button</tabstop>
  <tabstop>member_lwidget</tabstop>
  <tabstop>aimag_cbox</tabstop>
  <tabstop>down_aimag_button</tabstop>
  <tabstop>up_aimag_button</tabstop>
  <tabstop>aimag_lwidget</tabstop>
  <tabstop>soum_cbox</tabstop>
  <tabstop>down_soum_button</tabstop>
  <tabstop>up_soum_button</tabstop>
  <tabstop>soum_lwidget</tabstop>
  <tabstop>help_button</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
